### PR TITLE
refactor: icon update

### LINKED ISSUE #3164: prepare release 0.4.0
### migration tool

We need to change some icon namings.

### PR DESCRIPTION
**Pull request by mfranzke**
## Proposed changes

a.o. Resolves #3164

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: packages/foundations/scripts/test/icon-q32024/has-changes.txt
Comment (mfranzke) [inline packages/foundations/scripts/test/icon-q32024/has-changes.txt:1]: @nmerget if we would split up this file, we would get a separate feedback for each of those replacements, wouldn't we? As well as adding a `no-change.txt`

Comment (mfranzke) [inline packages/foundations/scripts/test/icon-q32024/has-changes.txt:1]: @nmerget if we would split up this file, we would get a separate feedback for each of those replacements, wouldn't we? As well as adding a `no-change.txt`.

Comment (mfranzke) [inline packages/foundations/scripts/test/icon-q32024/has-changes.txt:1]: @nmerget probably I do have a misunderstanding here, but if I do put this files contents into `no-change.txt` and vice versa the test would still pass.

File: __snapshots__/foundations/chromium/Icons-should-match-screenshot-1/Icons-should-match-screenshot.png
Comment (mfranzke) [inline __snapshots__/foundations/chromium/Icons-should-match-screenshot-1/Icons-should-match-screenshot.png:1]: this screenshot still includes some olders icons.

Comment (nmerget) [reply __snapshots__/foundations/chromium/Icons-should-match-screenshot-1/Icons-should-match-screenshot.png:1]: fixed

**Review by nmerget — APPROVED** [review]

**Pull request merged**